src/transport_ofi.c

```diff
@@ -980,6 +981,38 @@ int query_for_fabric(struct fabric_info *info)
     return ret;
 }
 
+static int shmem_transport_ofi_target_ep_init() {
+    int ret = 0;
+
+    struct fabric_info* info = &shmem_transport_ofi_info;
+    info->p_info->ep_attr->tx_ctx_cnt = FI_SHARED_CONTEXT;
```

Comment:
I think this should be 0 for the target EP.

Reply:
Whoops, yup.